teams/Aragon One/2019.md

```diff
@@ -122,9 +113,50 @@ The Aragon Court is a service protocol designed to incentivize honest arbitratio
 
 ANT is intended to be used for governance and as the native currency for the network. In order to optimize ANT for these uses adjustments to ANT’s monetary policy may be necessary. This research would (1) explore the impact of creating a liquidity reserve and treasury bonds to tightly couple the networks assets with the valuation of ANT and (2) the feasibility of shifting the realization value appreciation to unit count rather than unit price using a scalar value token structure.
 
+### I09 – Caching on the Aragon client
+
+#### 🔜 Enhance aragonJS caching
+
+Make sure that aragonJS provides a much better caching mechanism, so the first load is slightly faster, and subsequent loads are much faster than they are now.
+
+#### 🔜 Experiment and implement centralized/decentralized hybrid for first-load
+
+Even enhancing aragonJS caching, the first load may still take a while. That's why it makes sense to experiment with centralized/decentralized hybrid solutions for it. For example, a trusted set of participants in the DAO could decide what the state was a at a given block, and then everyone would start loading from that. Then over time the client could sync locally and arrive to a fully decentralized/trustless state.
+
+### 🔜 I10 – Email notifications
+
+All actions on DAOs are irreversible and some of them may involve funds, so notifications become very important to the DAO's stakeholders.
+
+We will implement opt-in notifications using an email server and another server which will track all DAO-related events on Ethereum.
+
+### 🔜 I11 – Aragon client re-design and general enhancements
+
+We plan to re-design the Aragon client by creating a new Aragon Human Interface Guidelines that standardizes how to create Aragon apps.
+
+As part of this re-design, we will also re-design the following items, based on user feedback:
+
+- The Permissions app
+- The Voting app
+
+### 🔜 I12 – Support Aragon Funraising development
```

Comment:
```suggestion
### 🔜 I12 – Support Aragon Fundraising development
```

Reply:
I don't know how this got in again, thanks for noticing!